### DISCUSSION TITLE
Alerts not showing up

### DISCUSSION THREAD

**Discussion by nilssachs** [Security-Onion-Solutions/securityonion#11559] [2.4]
### Version

2.4.20

### Installation Method

Security Onion ISO image

### Description

other (please provide detail below)

### Installation Type

Distributed

### Location

on-prem with Internet access

### Hardware Specs

Meets minimum requirements

### CPU

8

### RAM

64

### Storage for /

500GB

### Storage for /nsm

3TB

### Network Traffic Collection

span port

### Network Traffic Speeds

1Gbps to 10Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

Hi again,

in my current setup, all Suricata alerts are not showing up in the Alert dashboards.
Within the dashboard I am getting tons of alerts from the sensors, but none of those are displayed as alerts (tags:alert).

`* AND event.dataset:"suricata" | groupby -sankey event.dataset event.category* | groupby -pie event.category | groupby -bar event.module* | groupby event.dataset | groupby event.module* | groupby event.category | groupby observer.name | groupby source.ip | groupby destination.ip | groupby destination.port | groupby destination_geo.organization_name`

<img width="764" alt="image" src="https://github.com/Security-Onion-Solutions/securityonion/assets/12412595/37ee65ad-5df1-4f99-9fd8-b571627ed72f">

Since I also can't find the rule.* fields in the logs, but getting the whole suricata log within the message, is there something going wrong in the event parsing?

Thanks for your support, Nils

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to Security Onion 2.4.

**Comment by nilssachs** [comment]
Found the issue. Excluded the metadata in a custom Logstash Pipeline.
I know, 3rd party, but does anyone have an idea, why the mutation is done not just for the single pipeline, but obviously for all of them?

Edit: Pipeline is added for the Searchnode

**Comment by cm-ops** [comment]
What does you custom pipeline look like?
